utils/api/benefitHandler.ts

```diff
@@ -991,6 +991,17 @@ export class BenefitHandler {
             '--- both are not eligible for alw - applicant oas = 0 & partner oas > 0 --- end'
           )
         }
+        if (
```

Comment:
good this works, but move the code to before line 990, if you look at the console messages you'll see why.